perk-store-webapps/src/main/webapp/css/main.less

```diff
@@ -22,10 +22,12 @@ Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
     background: white;
 
     .toolbarOrders {
-      margin-bottom: 17px;
+      padding: 15px 5px 15px 15px;
```

Comment:
delete this please and use class of vuetify 
padding: 15px 5px 15px 15px; ===> pa-4